community/command-line/src/main/java/org/neo4j/commandline/admin/AdminTool.java

```diff
@@ -36,9 +36,9 @@
     public static final int STATUS_SUCCESS = 0;
     public static final int STATUS_ERROR = 1;
     // These fields are mutable for in-process AdminTool testing purposes.
-    public static String NEO4J_HOME = System.getenv().getOrDefault( "NEO4J_HOME", "" );
-    public static String NEO4J_CONF = System.getenv().getOrDefault( "NEO4J_CONF", "" );
-    public static String NEO4J_DEBUG = System.getenv().getOrDefault( "NEO4J_DEBUG", "" );
+    public static final String NEO4J_HOME = System.getenv().getOrDefault( "NEO4J_HOME", "" );
+    public static final String NEO4J_CONF = System.getenv().getOrDefault( "NEO4J_CONF", "" );
+    public static final String NEO4J_DEBUG = System.getenv().getOrDefault( "NEO4J_DEBUG", "" );
```

Comment:
These are mutable for testing, and exploited in the blockdevice repo, I believe.

Reply:
The comment just above also says so.

Reply:
Well spotted. Although, why can't the affected tests instantiate `AdminTool` directly as all other tests do?

Reply:
Maybe that wasn't possible at that time.

Reply:
Should hopefully be possible now, I'll have a look at it.